docs/section21/Rule21-2.md

```diff
@@ -0,0 +1,62 @@
+
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** For purchased HW or steam loops in the proposed design, baseline should include all on-site distribution pumps included in the proposed design.  
+**Rule Assertion:** B-RMR LEN(ASHRAE229:conditioning_component.pump) = P-RMR LEN(ASHRAE229:conditioning_component.pump) for pumps serving purchased hot water or steam loops  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.3.4 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is modeled with purchased hot water or steam
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+## Rule Logic:  
+
+- For each conditioning component in P_RMR: `for conditioning_component_p in P_RMR.ASHRAE229.conditioning_components:`
+
```

Comment:
I do not see conditioning_component in the schema - can you please point me in the right direction?